services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/LinkOperationsBolt.java

```diff
@@ -63,6 +64,9 @@
     private transient LinkOperationsService linkOperationsService;
     private transient FlowOperationsService flowOperationsService;
 
+    private static LinkPropsRepository linkPropsRepository;
+    private static IslRepository islRepository;
```

Comment:
Please make it `private transient` (without static). Just like services above

Reply:
Fixed